### PR TITLE
The project can now read from files!! :tada:

### PR DESCRIPTION
**Pull request by cawolfkreo**
# Description
This is still some basic file reading usage. Basically, we can now read the CPU commands from a file instead of using the `const` we started with. That constant is going to be removed at some point and I expect much of this code is going to change as well, but now, at least we can start getting more serious with all the instructions we implement for the CPU.

## How to run with the file:
Basically, you only need to supply a path for `cargo` in the parameters for the executable. I added the `data/test.asm` file. If we want to use it then we would simply run the program like this:

```Bash
cargo run -- data/test.asm
```

the previous way run the project still works and we still have the original `const` in case we don't want to use the file and whatnot.

### CODE REVIEW

**Review by afBejarano — APPROVED** [review]
Nice file reading
